src/eth_rpc/servers/debug_rpc.rs

```diff
@@ -60,7 +59,27 @@ impl<P: EthereumProvider + Send + Sync + 'static> DebugApiServer for DebugRpc<P>
     }
 
     /// Returns an array of EIP-2718 binary-encoded receipts.
-    async fn raw_receipts(&self, _block_id: BlockId) -> Result<Vec<Bytes>> {
-        Err(EthApiError::Unsupported("debug_rawReceipts").into())
+    async fn raw_receipts(&self, block_id: BlockId) -> Result<Vec<Bytes>> {
+        Ok(self
+            .eth_provider
+            .block_receipts(Some(block_id))
+            .await?
+            .unwrap_or_default()
+            .into_iter()
+            .map(|receipt| {
+                Receipt {
+                    tx_type: receipt.transaction_type.to::<u8>().try_into().unwrap(),
+                    success: receipt.status_code.unwrap_or_default().to::<u8>() == 1,
+                    cumulative_gas_used: receipt.cumulative_gas_used.to::<u64>(),
```

Comment:
Same, I would in general avoid using `to` unless when converting from `U_n` to `u_n`.

Reply:
I've used the same approach as before here, with the error handled into a conversion error item.

Reply:
Maybe indeed, misread the code